R/unnecessary_placeholder_linter.R

```diff
@@ -33,7 +33,7 @@
 unnecessary_placeholder_linter <- function() {
   # TODO(michaelchirico): handle R4.2.0 native placeholder _ as well
   xpath <- "
-  //SPECIAL[text() = '%>%']
+  //SPECIAL[text() = '%>%' or text() = '%T>%' or text() = '%<>%']
```

Comment:
Note which are covered in the documentation?

Reply:
Good point. Done.

We can revise this once the native pipe is also supported.